src/vm/eventpipebuffermanager.cpp

```diff
@@ -883,6 +812,18 @@ void EventPipeBufferList::SetOwnedByThread(bool value)
     m_ownedByThread = value;
 }
 
+bool EventPipeBufferList::GetThreadEventWriteInProgress()
```

Comment:
nit: make this method `const`: `bool EventPipeBufferList::GetThreadEventWriteInProgress() const`

Reply:
thanks :)